src/Microsoft.Identity.Client/Cache/CacheFallbackOperations.cs

```diff
@@ -254,44 +252,26 @@ internal static class CacheFallbackOperations
                         p.Key.ClientId.Equals(clientId, StringComparison.OrdinalIgnoreCase) &&
                         environmentAliases.Contains(new Uri(p.Key.Authority).Host)).ToList();
 
-                //if client info is provided then use it to filter
-                if (!string.IsNullOrEmpty(rawClientInfo))
-                {
-                    List<KeyValuePair<AdalTokenCacheKey, AdalResultWrapper>> clientInfoEntries =
-                        listToProcess.Where(p => rawClientInfo.Equals(p.Value.RawClientInfo, StringComparison.OrdinalIgnoreCase)).ToList();
-                    if (clientInfoEntries.Any())
-                    {
-                        listToProcess = clientInfoEntries;
-                    }
-                }
-
                 //if upn is provided then use it to filter
```

Comment:
can you give some context into why you're removing this and the part on uniqueId?

Reply:
rawClientInfo is being removed because we only call this method in one place, and always pass null.  (see previous TokenCache.cs around line 873 where it's called)

The HomeAccountId removal is wrong since there are cases where adal users have clientinfo and then this is needed.  will send out another fix.